content/account/users.textile

```diff
@@ -74,4 +74,27 @@ To remove a user from your account:
 
 h2(#delete). Delete your account
 
-Follow "this process":https://faqs.ably.com/how-do-i-cancel-my-account to delete your Ably account. Only an account owner can initiate this process.
+You can cancel your account at any time by downgrading to the "Free package":/docs/pricing/free. The following steps outline the the process to cancel your account:
+
+* Ensure you are the account "owner":/docs/account/users#roles.
+* Downgrade all paid "packages":/docs/pricing#packages free packages:
+** Navigate to *Account* then *Billing*.
+** Click *Save billing details* to save your changes.
+* After all packages have been downgraded to *Free*, go to the *My Settings* page.
+** Navigate to *Want to delete your profile?*.
+** Click *start* to begin the closure process.
+* On the *Close Your Ably Account* page:
+** Review the account summary, including ownership, plan type, and app details.
+** Confirm all accounts are on the *Free* tier and details are accurate.
+** Click *Confirm Closure* to permanently deactivate your Ably account.
+
+If you sign in with SSO, for example, Google or GitHub, you cannot use the self-service account closure process. To proceed, you must set a password and disconnect their SSO provider via My Settings.
+
+* Log in to your Ably account using your current SSO method (Google or GitHub).
+* Navigate to *Account* then *My Settings*.
+* In the *Password* section, click *Change your password*.
+* Click *Update my personal settings* to save the changes.
+* Scroll to the *Login provider* section.
+* Click *remove connection* next to the SSO provider/s you want to disconnect.
```

Comment:
I'm assuming after you do these steps, you can then close your account as per above? If that's the case, we need a final bullet point here to note that, and also to update the paragraph that states 'you cannot use the self-service account closure'.